lib/Swagger/src/DesignSystem/Api/DSApi.php

```diff
@@ -0,0 +1,202 @@
+     * @param string $lang_id Language (required)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return string
+     */
+    public function getPageHeader($wiki_id, $lang_id)
+    {
+        list($response) = $this->getPageHeaderWithHttpInfo($wiki_id, $lang_id);
+        return $response;
+    }
+
+    /**
+     * Operation getPageHeaderWithHttpInfo
+     *
+     * Returns page header data for the specified wikiId and langId
+     *
+     * @param int $wiki_id Wikia ID (required)
+     * @param string $lang_id Language (required)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return array of string, HTTP status code, HTTP response headers (array of strings)
+     */
+    public function getPageHeaderWithHttpInfo($wiki_id, $lang_id)
+    {
+        // verify the required parameter 'wiki_id' is set
+        if ($wiki_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $wiki_id when calling getPageHeader');
+        }
+        // verify the required parameter 'lang_id' is set
+        if ($lang_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $lang_id when calling getPageHeader');
+        }
+        // parse inputs
+        $resourcePath = "/wikis/{wiki_id}/{lang_id}/page-header";
+        $httpBody = '';
+        $queryParams = [];
+        $headerParams = [];
+        $formParams = [];
+        $_header_accept = $this->apiClient->selectHeaderAccept(['application/json']);
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType([]);
+
+        // path params
+        if ($wiki_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "wiki_id" . "}",
+                $this->apiClient->getSerializer()->toPathValue($wiki_id),
+                $resourcePath
+            );
+        }
+        // path params
+        if ($lang_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "lang_id" . "}",
+                $this->apiClient->getSerializer()->toPathValue($lang_id),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
+        } elseif (count($formParams) > 0) {
+            $httpBody = $formParams; // for HTTP post (form)
+        }
+        // make the API Call
+        try {
+            list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
+                $resourcePath,
+                'GET',
+                $queryParams,
+                $httpBody,
+                $headerParams,
+                'string',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "string" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)